src/biokbase/narrative/widgetmanager.py

```diff
@@ -585,6 +585,8 @@ def show_data_widget(self, upa, title=None, cell_id=None, tag="release"):
                 }
             }
         else:
+            if not type_spec.get('view_method_ids'):
```

Comment:
This should cover the case where 'view_method_ids' doesn't exist, OR when it does and it's an empty array.